packages/zoe/src/contracts/multipoolAutoswap/getCurrentPrice.js

```diff
@@ -55,30 +130,13 @@ export const makeGetCurrentPrice = (isSecondary, isCentral, getPool) => {
    * @returns {Amount} The amount required to be paid in order to gain amountOut
    */
   const getInputForGivenOutput = (amountOut, brandIn) => {
-    const { brand: brandOut, value: outputValue } = amountOut;
-
-    if (isCentral(brandIn) && isSecondary(brandOut)) {
-      return getPool(brandOut).getCentralToSecondaryOutputPrice(outputValue);
-    }
-
-    if (isSecondary(brandIn) && isCentral(brandOut)) {
-      return getPool(brandIn).getSecondaryToCentralOutputPrice(outputValue);
-    }
-
-    if (isSecondary(brandIn) && isSecondary(brandOut)) {
-      // We must do two consecutive calls to get the price: from
-      // the brandIn to the central token, then from the central
-      // token to the brandOut.
-      const centralTokenAmount = getPool(
-        brandIn,
-      ).getSecondaryToCentralOutputPrice(outputValue);
-      return getPool(brandOut).getCentralToSecondaryOutputPrice(
-        centralTokenAmount.value,
-      );
-    }
-
-    throw new Error(`brands were not recognized`);
+    return getPriceGivenRequiredOutput(brandIn, amountOut).amountIn;
```

Comment:
I'm just realizing this, but the inconsistency of the ordering of the arguments between `getInputForGivenOutput` and `getPriceGivenRequiredOutput` is going to be a problem for users. It doesn't have to be in this PR, but we need to resolve this inconsistency. Let's at least make an issue for this if there isn't already one.

Reply:
I'm make an issue. The new methods have their params in the in-then-out order, which I'd like to use for all, but I'm not sure what the migration path should be.

Reply:
https://github.com/Agoric/agoric-sdk/issues/2267